components/security-mgt/org.wso2.carbon.security.mgt.ui/src/main/resources/web/securityconfig/view-scenario-detail_ajaxprocessor.jsp

```diff
@@ -19,8 +19,14 @@
 <%
     String scenarioIdNumber = "";
     String scenarioId = request.getParameter("scenarioId");
+    try {
+        Integer.parseInt(scenarioId);
+    } catch (Exception e) {
```

Comment:
Since we don't need to catch and handle specific exceptions like number format exception, it's OK to catch Exception.